content/learn/book/migration-guides/0.5-0.6/_index.md

```diff
@@ -273,6 +273,37 @@ let xyz: IVec3 = xyz.as_ivec3();
 The {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.6.0" name="StandardMaterial" no_mod=true)}} field `roughness` was renamed to `perceptual_roughness`.
 
 ### Visible is now Visibility 
-The {{rust_type(type="struct" crate="bevy" mod="render::draw" version="0.5.0" name="Visible" no_mod=true)}} struct, which is used in a number of components to set visibility, was renamed to {{rust_type(type="struct" crate="bevy" mod="render::view" version="0.6.0" name="Visibility" no_mod=true)}}
+The {{rust_type(type="struct" crate="bevy" mod="render::draw" version="0.5.0" name="Visible" no_mod=true)}} struct, which is used in a number of components to set visibility, was renamed to {{rust_type(type="struct" crate="bevy" mod="render::view" version="0.6.0" name="Visibility" no_mod=true)}}. Additionally, the field `is_transparent` was removed from the struct. For 3D, transparency can be set using the `alpha_mode` field on a material. In 2D, everything is transparent. 
 
-Additionally, the field `is_transparent` was removed from the struct. For 3D, transparency can be set using the `alpha_mode` field on a material. In 2D, everything is transparent. 
+
+```rust
+// 0.5
+let material_handle = materials.add(StandardMaterial {
+                          base_color_texture: Some(texture.clone()),
```

Comment:
What's going on with the indentation here?

Reply:
Looks good, except for the indentation like @rparrett already mentioned.
I would change the indentation to the following:

```rust
    let material_handle = materials.add(StandardMaterial {
        base_color_texture: Some(texture.clone()),
        ..Default::default()
    });
```

```rust
    let material_handle = materials.add(StandardMaterial {
        base_color_texture: Some(texture.clone()),
        alpha_mode: AlphaMode::Blend,
        ..Default::default()
    });
```